internal/v4/api_test.go

```diff
@@ -2920,3 +2929,5 @@ func (s *APISuite) TestSetAuthCookieMethodError(c *gc.C) {
 		},
 	})
 }
+
+var _ entitycache.Store = v5.ChannelStore{}
```

Comment:
Not strictly necessary - the type is verified by being passed to entitycache.New.